city_scrapers/spiders/il_port_district.py

```diff
@@ -0,0 +1,183 @@
+                    start=start,
+                    end=None,
+                    all_day=False,
+                    time_notes="",
+                    location=self.location,
+                    links=links,
+                    source=response.url
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def parse_agendas(self, response):
+        file_names = response.xpath("//tr/td[@class='views-field views-field-title']/text()")\
+            .extract()
+        file_names = [x.strip("\n ") for x in file_names]
+        file_links = response.xpath("//tr/td/a[@class='file-download']/@href").extract()
+
+        self.files_list = []
+
+        for link, name in list(zip(file_links, file_names)):
+            self.files_list.append({'title': name, 'href': link})
+
+    def parse_minutes(self, response):
+        rows = response.xpath("//tr")
+        self.minutes_dict = {}
+        for row in rows:
+            file_name = row.xpath(".//td[@class='views-field views-field-title']/text()")\
+                .extract_first()
+            if not file_name:
+                continue
+
+            file_name = file_name.strip("\n ")
+            file_name_dt = re.findall(r"(?:.*?)(?:\d{4})", file_name)[0]
+            file_name_dt = datetime.datetime.strptime(file_name_dt, "%B %d, %Y")
+
+            file_link = row.xpath(".//td[@class='views-field views-field-field-file']/a/@href")\
+                .extract_first()
+
+            self.minutes_dict.setdefault(file_name_dt.date(), file_link)
+
+    def _parse_classification(self, i, meeting_types):
+        if "board" in meeting_types.lower():
+            return BOARD
+        elif "committee" in meeting_types.lower():
+            return COMMITTEE
+        else:
+            return NOT_CLASSIFIED
+
+    def _parse_start(self, year, date, meeting_time):
+        dt = " ".join([year, date, meeting_time])
+        return datetime.datetime.strptime(dt, "%Y %B %d %I:%M%p")
+
+    def _parse_agendas_links(self, classification):
+        if classification == 'Board':
+            link_list = [d for d in self.files_list if 'Board' in d['title']]
+        elif classification == 'Committee':
+            link_list = [d for d in self.files_list if 'Committee' in d['title']]
+        else:
+            link_list = []
+
+        return link_list
+
+    def _parse_minutes_links(self, start):
+        link = self.minutes_dict.get(start.date(), None)
+        if link:
+            name = "Board Meeting Minutes"
+            return [{'title': name, 'href': link}]
+        else:
+            return None
+
+    def _validate_location(self, response):
+        loc = response.xpath("//strong")[-1].xpath(".//text()").extract()
+        loc = [x.strip("\n ") for x in loc]
+        loc = " ".join(loc[-2:])
+        if '3600 E. 95th St. Chicago, IL 60617' != loc:
```

Comment:
To make sure we're not raising an error if they remove a period, we could just check for `3600` here